src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.ts

```diff
@@ -97,9 +97,9 @@ export class GeneNominatedTargetsComponent implements OnInit {
         // First map all entries nested in the data to a new array
         if (de.target_nominations?.length) {
           teamsArray = de.target_nominations.map((nt: TargetNomination) => nt.team);
-          studyArray = de.target_nominations.map(
-            (nt: TargetNomination) => nt.study
-          );
+          studyArray = de.target_nominations
+            .map((nt: TargetNomination) => nt.study)
+            .filter((item) => Boolean(item)) as string[];
```

Comment:
Fix for AG-1328 -- drop study values that are null or empty strings. See Cohort Study column below.

develop: 
<img width="1486" alt="AG-1328_study_develop" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/1d5624ad-1aab-4551-9305-3af6afbff8cf">

feature: 
 <img width="1483" alt="AG-1328_study_feature" src="https://github.com/Sage-Bionetworks/Agora/assets/26949006/62830c4a-87e3-4c34-92cb-9ba29cbe3fd9">

Reply:
as a suggestion, perhaps we can create a helper method to remove null/empty values .

Reply:
Done in the next commit